website/docs/plugins/optuna_sweeper.md

```diff
@@ -6,7 +6,7 @@ sidebar_label: Optuna Sweeper plugin
 
 import GithubLink,{ExampleGithubLink} from "@site/src/components/GithubLink"
 
-[![PyPI](https://img.shields.io/pypi/v/hydra-optuna-sweeper)](https://img.shields.io/pypi/v/hydra-optuna-sweeper)
+[![PyPI](https://img.shields.io/pypi/v/hydra-optuna-sweeper)](https://pypi.org/project/hydra-optuna-sweeper/)
```

Comment:
If we decide to officially support Hydra 1.1 and newer here we should mention it early in the readme to avoid surprises by users installing the plugin and having their Hydra upgraded to 1.1.

Reply:
Thank you for your suggestion.
I set the version limitation of the hydra-core as `hydra-core>1.0` in the commit https://github.com/facebookresearch/hydra/pull/1294/commits/9b231efab9fe2dd347847f45d4da80810805f32d. And then, I explained the version limitation of Hydra in the installation section in https://github.com/facebookresearch/hydra/pull/1294/commits/d71aec8aec783312c1c2d8ae6ec291ae71e01588.